src/components/steps/FinishedStep.js

```diff
@@ -24,13 +25,17 @@ const FinishedStep = ({
   title,
   linkText,
   secondaryActions,
+  uuid,
 }) => (
   <Bullseye>
     <EmptyState variant={EmptyStateVariant.full} className="pf-u-mt-4xl">
       <EmptyStateIcon icon={CheckCircleIcon} color="var(--pf-global--success-color--100)" className="pf-u-mb-0" />
       <Title headingLevel="h2" size="xl" className="pf-u-mt-xl">
         {title}
       </Title>
+      <ClipboardCopy isReadOnly hoverTip="Source UUID" clickTip="Copied" className="pf-u-mt-md">
+        {uuid}
```

Comment:
Because the prop is not required can we do a check if the UUID exists first before we show the component?